src/workspace.coffee

```diff
@@ -550,7 +556,7 @@ class Workspace extends Model
   # Extended: Create a new text editor.
   #
   # Returns a {TextEditor}.
-  buildTextEditor: (params) ->
+  buildTextEditor_: (params) ->
```

Comment:
Yeah agreed, it's better if the object only has a `buildTextEditor` method, not both `buildTextEditor` and `buildTextEditor_`.